src/router/router.tsx

```diff
@@ -27,6 +28,7 @@ export const RouterBase = (): ReactElement => {
         <ToastContainer />
 
         <Switch>
+          <Route path="/demo" exact component={DemoPage} />
```

Comment:
Do we still need this?

Reply:
No, i will remove it after completing review.

Reply:
just yet another example for reviewers